project-api/project-api-test/src/test/resources/projects_for_resolution_tests/package_l_with_stable_dep/Ballerina.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+org = "samjs"
+name = "package_l"
+version = "2.1.0"
```

Comment:
since this version (2.1.0) and the package (package_l_with_stable_dep) is also same, won't they get overridden in the the balo cache location, when compiling balo?

Reply:
just wondering whether that could have an impact on the test cases

Reply:
It is not used in these test cases. I will delete them in my next PR. Thanks for catching this